### DISCUSSION TITLE
I cannot seem to opt in or opt out of nsfw downloads from the saved list

### DISCUSSION THREAD

**Discussion by user4302** [Serene-Arc/bulk-downloader-for-reddit#501] [Q&A] [answered]
i am unable to download posts without the nsfw tag, nor any posts only with the nsfw tag.

the script i use is this:
`python -m bdfr download D:\path\to\save\folder --user myUsername --saved --authenticate --file-scheme [{REDDITOR}]-[{DATE}]_{TITLE}_[{POSTID}]  --folder-scheme {SUBREDDIT} --no-dupes --search 'nsfw:0' --make-hard-links --search-existing`

is this valid only when downloading from a specific subreddit?
if not, how do i make this work?

**Comment by Serene-Arc** [comment, marked as answer]
No. So using the `--saved` option, anything applied in the `--search` option isn't applied. The search just won't affect it. If you think this should be possible, you can open an enhancement request and I'll get to it as soon as possible.
